openassessment/xblock/ui_mixins/legacy/handlers_mixin.py

```diff
@@ -2,6 +2,8 @@
 
 import logging
 
+from platform_plugin_turnitin.events.data import ORASubmissionData
+from platform_plugin_turnitin.events.signals import ORA_SUBMISSION_CREATED
```

Comment:
This is a non-blocking comment: I know we've discussed adding the plugin directly here, but what do you think about extracting the event/data into our openedx-events fork? Then, it'd be easier to contribute to the community. The changes would be moving the events to the fork and installing the opened-events fork here instead of the PyPI release. We can do it if it's not too much work.

Reply:
It makes sense what you propose. I'll make the corresponding changes, it won't take much work.